### ISSUE TITLE
matching bracket colors dont match in command output pane

### ISSUE THREAD

**Issue by ecterrab** [Komodo/KomodoEdit#223]
You run any command, it produces legible ASCII output in the lower pane. Great. You want now select a portion using the matching brackets facility. It works GREAT. But there is one issue: the colours used for matching brackets in the lower pane does not respect the choices set in the preferences. So in the main window of Komodo, matching brackets follow the choice set in the preferences, but not in the lower pane. Expected would be to have the lower pane respecting these choices. This problem is still there in Kmodo 9.0.1 build 15738; perhaps it is a simple thing to adjust? Thanks

imported from old database, #102211

**Comment by Naatan** [comment]
Could you provide a test case to reproduce the issue?

**Comment by ecterrab** [comment]
I am attaching here two images showing the issue, but the way to reproduce the problem requires some commands that I do not have at hands, would need to think ... Anyway: I am running a UNIX command and receive the output in the console. Great. The output contains some matching brackets. The color when matching the brackets does not respect the colors I set in the preference, that are respected - work fine - in the main window.

In the attached image you see the matching bracket color I set in the preferences to blue in bold, in line 31, around the word 'signature', while in the lower pane the matching brackets appears in a sort of maroon light (Komodo's default, I suppose), for the square brackets also around the word signature.

Expected: also the preference setting blue/bold for the matched brackets in the lower pane.
![screen shot 2015-04-27 at 11 51 20 am](https://cloud.githubusercontent.com/assets/12106224/7355085/81a2b5ce-ecd4-11e4-9621-82125d28847d.png)


I note as well that the lower (bottom) filed, displayed in grey, should be displayed in black and the fonts in grey, so that you can read the actual command I ran to generate the output in the lower pane. In this moment the background and font colors are the same, grey, so you cannot read anything. I tracked this as another issue, with number #213 . In the following image, I clicked with the mouse somewhere else outside Komodo, to trigger another bug, also tracked in #213, where through this clicking elsewhere action the color of the foreground tab titles (both in the main window and the lower pannel), change unexpectedly, and also that the bottom so that now you can see that the UNIX command that generated this output in the lower pane is actually a sed command with some instructions related to my work:
![screen shot 2015-04-27 at 11 51 35 am](https://cloud.githubusercontent.com/assets/12106224/7355078/727aeada-ecd4-11e4-9f3b-914b94f68a10.png)

**Comment by Naatan** [comment]
Is that one of the default color schemes you're using? If not; are you able to reproduce the issue with a default color scheme?

**Comment by ecterrab** [comment]
I don't know the answer to your question. I don't see any default color scheme for 'Language Specific - Text". I set these colours years ago as shown in the two images attached:
![screen shot 2015-04-27 at 3 57 07 pm](https://cloud.githubusercontent.com/assets/12106224/7359733/c1770002-ecf6-11e4-9e6c-4dcacc74668d.png)
![screen shot 2015-04-27 at 3 57 07 pm](https://cloud.githubusercontent.com/assets/12106224/7359737/c6881aea-ecf6-11e4-8bd1-5ceac1a030da.png)

**Comment by ecterrab** [comment]
I experimented furthermore: if you change the "Scheme: (system schemes are in bold)" field to any one of the menu, the matching of brackets in the main window and the lower pane look the same - correct. But if you happen to change the color in any way, you are asked to save the new scheme with a name (I suppose that is what I did years ago, it end up called "New Default") and then it is not the default anymore. So apparently this happens when choosing a non-default color of bracehighlight.
